src/App.js

```diff
@@ -1,10 +1,47 @@
 import React from 'react';
 import './App.css';
+import handy from "./assets/bag_1.png";
+import stylish from "./assets/bag_2.png";
+import simple from "./assets/bag_3.png";
+import trendy from "./assets/bag_4.png";
+import brand from "./assets/brand.png";
+import story from "./assets/our_story.png";
+import Button from "./components/Buttons";
+import Products from "./components/Products";
+import Tiles from "./components/Tiles";
 
 function App() {
-  return (
-      <h1>Handbags & Purses</h1>
-  );
+    return (
+        <>
+            <h1>Handbags & Purses</h1>
+            <nav>
+                <Button buttonText="to the collection"/>
+                <Button buttonText="shop all bags"/>
+                <Button buttonText="pre-order"/>
+            </nav>
+            <main>
+                <Products advice="Best seller" productImage=<img src={handy} alt="Handy bag"/> articleName="The handy
+                bag" price="€450,-"/>
+                <Products advice="Best seller" productImage=<img src={stylish} alt="Stylish bag"/> articleName="The
+                stylish bag" price="€250,-"/>
+                <Products advice="New collection" productImage=<img src={simple} alt="Simple bag"/> articleName="The
+                simple bag" price="€300,-"/>
+                <Products advice="New collection" productImage=<img src={trendy} alt="Trendy bag"/> articleName="The
+                trendy bag" price="€150,-"/>
+            </main>
+            <footer>
+                <Tiles title="The brand" paragraph="Lorem ipsum dolor sit amet, consectetur adipisicing elit. Est itaque natus odit quaerat unde,
+                        voluptatum. Alias consequatur cupiditate dolore laborum minus nemo provident quas rerum."
+                />
+                <Tiles tileImage=<img src={brand} alt="Brand"/> />
```

Comment:
Je Tiles zijn inderdaad niet helemaal netjes. Dynamische content waarvan je niet zeker weet of het er komt zetten we in Children. 

Children staan tussen de opening en de closing tag. hiervoor moet je dus niet een self closing tag gebruiken voor je component. Children hebben verder ook geen naam. Je krijgt alle elementen tussen de opening en closing tag terug in een Array onder the variable Children die je tussen je vaste content kan plaatsen.